plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4265,6 +4267,101 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
                 }
                 return new ElementTag(chiseledBookshelf.getLastInteractedSlot() + 1);
             });
+
+            // <--[language]
+            // @name Structure lookups
+            // @description
+            // There are several tags to locate structures, such as <@link tag LocationTag.find_structure> and <@link tag LocationTag.find_structure_type>.
+            // These tags work similarly to the '/locate' command, and have several side effects/edge cases:
+            // - The radius is in chunks, but isn't always a set square radius around the origin; certain structures may modify the amounts of chunks checked.
+            //   For example, woodland mansions can potentially check up to 20,000 blocks away (or more) regardless of the radius used.
+            // - Lookups can take a long amount of time (several seconds, over 10 in some cases), especially when looking for unexplored structures,
+            //   which will cause the server to freeze while searching.
```

Comment:
this will look weird on meta site - just put it on one line

Reply:
In-lined it